src/Microsoft.AspNet.Mvc/MvcOptionsSetup.cs

```diff
@@ -62,11 +63,13 @@ public static void ConfigureMvc(MvcOptions options)
 
             // Set up metadata providers
             options.ModelMetadataDetailsProviders.Add(new DefaultBindingMetadataProvider());
+            options.ModelMetadataDetailsProviders.Add(new DefaultValidationMetadataProvider());
             options.ModelMetadataDetailsProviders.Add(new DataAnnotationsMetadataDetailsProvider());
```

Comment:
see this type is still called `DataAnnotationsMetadataDetailsProvider` ...and not `DataAnnotationsMetadataProvider`